fastlane_bot/events/utils.py

```diff
@@ -617,6 +621,13 @@ def get_config(
             self_fund=self_fund,
         )
         cfg.logger.info("[events.utils.get_config] Using mainnet config")
+
+    if rpc_url:
+        cfg.w3 = Web3(Web3.HTTPProvider(rpc_url, request_kwargs={"timeout": 60}))
+        cfg.w3_async = AsyncWeb3(AsyncWeb3.AsyncHTTPProvider(rpc_url))
+        if 'tenderly' in rpc_url:
+            cfg.NETWORK = cfg.NETWORK_TENDERLY
```

Comment:
This looks suspicious, with respect to:
```
network = self.NETWORK if "tenderly" not in self.NETWORK else "ethereum"
```
In file `network.py`.
Can you please verify that there is no "collision" between these two?

Reply:
I can confirm there is no collision between the two, however your suspicions are correctly identifying a somewhat 'hacky' solution. Unfortunately the convoluted state of the config/* folder dictates my approach here, and I suggest we address the underlying issue separately ASAP. For the current purpose however, I can confirm there is no collision.